cypress/e2e/selectors/constants.cy.ts

```diff
@@ -19,6 +19,8 @@ export const TOKEN_SYMBOLS = {
     'BNB': ['RICE', 'DAI', 'USDT', 'USDC', 'MAI']
 }
 
+export const NETWORK_LIST = ['Ethereum', 'Arbitrum', 'Optimism', 'Avalanche', 'BNB Chain', 'Polygon PoS', 'Fantom', 'Linea', 'Base']
```

Comment:
how about call api to get it. https://ks-setting.kyberswap.com/api/v1/configurations/fetch?serviceCode=chains ?

Reply:
Just need a few chains.